pkg/cloudprovider/cloudprovider.go

```diff
@@ -60,6 +61,7 @@ const (
 func init() {
 	v1alpha5.NormalizedLabels = lo.Assign(v1alpha5.NormalizedLabels, map[string]string{"topology.ebs.csi.aws.com/zone": v1.LabelTopologyZone})
 	coreapis.Resources = lo.Assign(coreapis.Resources, apis.Resources)
+	coreapis.Settings = coreapis.Settings.Union(apis.Settings)
```

Comment:
musing: I'd love to find a better path than this global injection, but this is fine for now.

Reply:
I'm curious why you think this global injection is bad? Scheme registration is done through the same mechanism and I'd argue these are very close concepts.

Reply:
Injection hides where values come from. Sometimes this is good (encapsulation) sometimes it's bad (obfuscation). I'm fine with it here, but it feels a touch obfuscated to me.